Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -212,7 +212,7 @@ and/or `throws` and to be isolated to a global actor (for example, by using the
 For more information about test functions and how to declare and customize them,
 see <doc:DefiningTests>.
 
-#### Converting XCTAssert() XCTUnwrap(), and XCTFail() calls
+#### Convert XCTAssert() XCTUnwrap(), and XCTFail() calls
```

Comment:
Delete; can't have subheadings within a section. Also, don't include symbols in headings.

Reply:
Consider breaking it into a separate section, then? The section gets quite long without it and some natural breaks from headings would help.